packages/layout/src/primitives/parsePanelsState.ts

```diff
@@ -89,7 +89,7 @@ export function parsePanelsState(currentlyActivePanel: string | undefined, panel
 				if (difference !== 0) {
 					return difference
 				} else {
-					difference = (b.priority ?? -1) - (a.priority ?? -1)
+					difference = numberOrFallback(b.priority, -1) - numberOrFallback(a.priority, -1)
```

Comment:
same here, priority can be either number or undefined /  null, right?

Reply:
Done with `||`